### DISCUSSION TITLE
WebBrowser1.Stop() -> cromiumBrowser.Stop()?

### DISCUSSION THREAD

**Discussion by Mephisto090** [cefsharp/CefSharp#4049] [Q&A] [answered]
Hello dear developer of Cefsharp,

is there a function that cancels loading like the IE browser?

`WebBrowser1.Stop()`

**Comment by amaitland** [comment, marked as answer]
http://cefsharp.github.io/api/99.2.x/html/M_CefSharp_WebBrowserExtensions_Stop_1.htm

Make sure to add a using statement for the CefSharp namespace.
